src/matlab2tikz.m

```diff
@@ -2906,13 +2914,18 @@ case size(cData,2) % specific y-data is given
 % ==============================================================================
 function [m2t, opts, s] = shaderOptsSurfPatch(m2t, handle, opts, s)
 
-            
     % Set opacity if FaceAlpha < 1 in MATLAB
     s.faceAlpha = get(handle, 'FaceAlpha');
     if isnumeric(s.faceAlpha) && s.faceAlpha ~= 1.0
-        opts = opts_add(opts,'opacity',sprintf(m2t.ff,s.faceAlpha));
```

Comment:
`opacity` alone applies to both `fill` and `draw`. Thus I changed it to `fill opacity` and additionally introduced `draw opacity`.